security-analysis/security-analysis-api/src/main/java/com/powsybl/security/operator/strategy/OperatorStrategyList.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.operator.strategy;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+public class OperatorStrategyList {
```

Comment:
Is this class needed?

Reply:
yes for the file needed for the securityAnalysisTool